manifests/charts/gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -266,8 +273,18 @@ spec:
             readOnly: true
           {{- end }}
           {{- end }}
-{{- if $gateway.additionalContainers }}
-{{ toYaml $gateway.additionalContainers | indent 8 }}
+{{- $noSGX := true }}
```

Comment:
Where is this noSGX variable coming from?

Reply:
Hi @poussa, L#276 is the source of `noSGX` variable.